src/argilla/client/sdk/client.py

```diff
@@ -96,6 +100,7 @@ def __post_init__(self):
             cookies=self.get_cookies(),
             timeout=self.get_timeout(),
         )
+        # TODO: Remove this NOW!!!!
```

Comment:
Is this an upcoming TODO or something to be tackled within this PR? 😆

Reply:
I will tackle in the next one, since we have too many changes here